ecs_composex/compose/compose_services/__init__.py

```diff
@@ -754,6 +754,25 @@ def ulimits(self) -> Union[list, Ref]:
 
         return rendered_limits if rendered_limits else NoValue
 
+    @property
+    def stop_grace_period(self):
+        """
+        Method to import and determine StopTimeout
+        """
+        __stop_grace_period = set_else_none("stop_grace_period", self.definition)
+        if not __stop_grace_period:
+            return NoValue
+        if not isinstance(__stop_grace_period, (int, str, float)):
```

Comment:
According to https://github.com/compose-spec/compose-spec/blob/master/schema/compose-spec.json#L356 the type has to be a string.
Given compose-x uses the spec to do validation, we probably will never get any other value than a string, but certainly shouldn't.
```suggestion
        if not isinstance(__stop_grace_period, (str)):
```